mapie/utils.py

```diff
@@ -160,15 +161,17 @@ def check_cv(
             return LeaveOneOut()
         if cv >= 2:
             return KFold(n_splits=cv)
-    if (
-        isinstance(cv, BaseCrossValidator)
-        or (cv == "prefit")
-        or (cv == "split")
-    ):
+    if isinstance(cv, BaseCrossValidator):
+        return cv
+    if isinstance(cv, BaseShuffleSplit):
+        return cv
+    if cv in ["prefit"]:
```

Comment:
Why using in List instead of == str ?

Reply:
There is no particular reason. I can change that with a string comparison if you think it is more pertinent. Let me know in order to add this change.

Reply:
I think `==` is better